doc/users/gdb.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+                 "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta http-equiv="Content-type" content="text/html;charset=UTF-8">
+  <title>deal.II GDB documentation</title>
+  <link href="../screen.css" rel="StyleSheet">
+  <meta name="author" content="the deal.II authors">
+  <meta name="copyright" content="Copyright (C) 2012 - 2015 by the deal.II authors">
+  <meta name="keywords" content="deal dealii finite elements fem triangulation">
+  <meta http-equiv="content-language" content="en">
+</head>
+<body>
+
+  <h1>Configuration for debugging deal.II projects via GDB</h1>
+
+  <p><acronym>deal.II</acronym> comes with a simple set of pretty-printers that
+  provide descriptions to <code>gdb</code> (the GNU debugger) on how to usefully
+  print <acronym>deal.II</acronym> objects. For example, without
+  pretty-printing, <code>gdb</code> prints points as</p>
+<pre class="sample">
+(gdb) print point
+$1 = {&lt;dealii::Tensor&lt;1, 2, double&gt;&gt; = {static dimension = &lt;optimized out&gt;,
+    static rank = &lt;optimized out&gt;, static n_independent_components = &lt;optimized out&gt;, values = {{
+        static dimension = &lt;optimized out&gt;, static rank = &lt;optimized out&gt;,
+        static n_independent_components = &lt;optimized out&gt;, value = 0}, {
+        static dimension = &lt;optimized out&gt;, static rank = &lt;optimized out&gt;,
+        static n_independent_components = &lt;optimized out&gt;, value = 0}}}, &lt;No data fields&gt;}
+</pre>
+  but with pretty-printing enabled GDB prints
+<pre class="sample">
+(gdb) print point
+$1 = {0, 0}
+</pre>
+  which is much easier to read.</p>
+
+  <p>Setting up support for <code>gdb</code> is simple: <code>gdb</code> looks
+  for the file <code>.gdbinit</code> in a user's home directory for pretty
+  printers, so one can simply copy the <acronym>deal.II</acronym> implementation
+  to that location with the following shell commands
+  (where <code>dealii_source_directory</code> is the root directory containing
+  all <acronym>deal.II</acronym> source files):
+<pre class="sample">
+  cd dealii_source_directory
+  cp contrib/utilities/dotgdbinit.py ~/.gdbinit
```

Comment:
This assumes that a user does not yet have such a file. Could we explain here (as well as the top of the dotgdbinit file) what to do if such a file already exists?